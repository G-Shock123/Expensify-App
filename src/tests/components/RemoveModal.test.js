import {shallow} from 'enzyme'
import RemoveModal from '../../components/RemoveModal'
import React from 'react'



let wrapper, handleNoChange, handleConfirmRemove

beforeEach(() => {
    wrapper = shallow(<RemoveModal handleConfirmRemove={handleConfirmRemove} handleNoChange={handleNoChange} selectedOption={true} />)
    handleNoChange = jest.fn()
    handleConfirmRemove= jest.fn()
})



test('Render remove modal',()=>{
    const wrapper = shallow(<RemoveModal />)
    expect(wrapper).toMatchSnapshot()
})

