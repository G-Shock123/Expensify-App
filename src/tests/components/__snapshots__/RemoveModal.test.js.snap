// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render remove modal 1`] = `
<div>
  <Modal
    ariaHideApp={true}
    bodyOpenClassName="ReactModal__Body--open"
    className="modal"
    closeTimeoutMS={200}
    contentLabel="Remove Modal"
    isOpen={false}
    parentSelector={[Function]}
    portalClassName="ReactModalPortal"
    role="dialog"
    shouldCloseOnEsc={true}
    shouldCloseOnOverlayClick={true}
    shouldFocusAfterRender={true}
    shouldReturnFocusAfterClose={true}
  >
    <div>
      <h3>
         Are you sure you want to remove the expense?
      </h3>
      <button
        className="button button--alert"
      >
        Yes 
      </button>
      <button
        className="button"
      >
        No
      </button>
    </div>
  </Modal>
</div>
`;
